### PR TITLE
ENH first version

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# make sure the install below is not cached by docker
+ADD http://worldclockapi.com/api/json/utc/now /opt/docker/etc/timestamp
+
+# Install conda
+RUN echo "**** install dev packages ****" && \
+    apk add --no-cache bash ca-certificates wget && \
+    \
+    echo "**** get Miniconda ****" && \
+    mkdir -p "$CONDA_DIR" && \
+    wget "http://repo.continuum.io/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh" -O miniconda.sh && \
```
Comment (scopatz) [inline Dockerfile:27]: This could use miniforge

Comment (beckermr) [reply Dockerfile:27]: link?

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2020, conda-forge
+Copyright (c) 2020, conda-forge (@cconda-forge/core)
```
Comment (scopatz) [inline LICENSE]: isn't it `@conda-forge/core`... I think there are two 'c's at the start

Comment (beckermr) [reply LICENSE]: woops

File: Dockerfile (hunk 1 above)
Comment (scopatz) [reply Dockerfile:27]: https://github.com/conda-forge/miniforge

Comment (beckermr) [reply Dockerfile:27]: ah cool - i'm gonna punt on this for now but will come back later

Comment (scopatz) [reply Dockerfile:27]: OK

**Comment by beckermr** [comment]
@isuruf can you take a look here? For whatever reason the token they give us is not allowing me to push to forks.

**Comment by isuruf** [comment]
After reading https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-events-for-forked-repositories-2 it looks like it is expected. Maybe we should ask github if they can extend the permissions to write on fork repo branch in addition to read on base repo.

**Comment by beckermr** [comment]
Right but I assumed that if I had "Allow edits from maintainers." checked, then it would let me write.

**Comment by beckermr** [comment]
Something is even weirder here though. I cannot even comment on a PR in the same repo.

**Comment by beckermr** [comment]
OK so it works for non-forked PRs. I don't quite understand how I didn't hit this issue with status and check_suite events, but I guess these are always sent to the default branch. The docs say pull requests go to the base repo as well, but I guess that is not quite what they mean. 

This PR will have to be on ice until the permissions can be changed. Certain pull requests events like labels should have write permissions since only maintainers can create labels in the first place.

File: rerender.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        retry=urllib3.util.retry.Retry(total=10, backoff_factor=0.1))
+
+    return gh
+
+
+def rerender(git_repo):
+    LOGGER.info('rerendering')
+    curr_head = git_repo.active_branch.commit
+    ret = subprocess.call(
+        ["conda", "smithy", "rerender", "-c", "auto"],
```
Comment (isuruf) [inline rerender.py]: Technically, following should same some time. (But the way it is now is that it just warns after making the solve, but we can fix that in conda-smithy)
```suggestion
        ["conda", "smithy", "rerender", "-c", "auto", "--no-check-uptodate"],
```

**Comment by isuruf** [comment]
Looks good to me. Thanks for working on this.

Btw, as this is now using `repository_dispatch` events, can we change the name of this action to something like, `webservice-dispatch-action` so that we can add more functionality if necessary later on.

**Comment by beckermr** [comment]
I'm going to merge and then swap the names in another PR.

**Pull request merged**